pkg/webhook/common.go

```diff
@@ -113,13 +113,13 @@ func (h *webhookHandler) getConfig(ctx context.Context) (*v1alpha1.Config, error
 
 // isGatekeeperResource returns true if the request relates to a gatekeeper resource.
 func (h *webhookHandler) isGatekeeperResource(req *admission.Request) bool {
-	if req.AdmissionRequest.Kind.Group == "templates.gatekeeper.sh" ||
-		req.AdmissionRequest.Kind.Group == "constraints.gatekeeper.sh" ||
-		req.AdmissionRequest.Kind.Group == mutationsGroup ||
-		req.AdmissionRequest.Kind.Group == "config.gatekeeper.sh" ||
-		req.AdmissionRequest.Kind.Group == externalDataGroup ||
-		req.AdmissionRequest.Kind.Group == "expansion.gatekeeper.sh" ||
-		req.AdmissionRequest.Kind.Group == "status.gatekeeper.sh" {
+	if req.Kind.Group == "templates.gatekeeper.sh" ||
```

Comment:
same issue

Reply:
`AdmissionRequest` is embedded field here.